anitya/templates/search.html

```diff
@@ -88,6 +88,7 @@
     <blockquote>
         Oups this is embarrassing, it seems that no projects are being
         monitored currently.
+        <p><a href="/project/new?name={{ pattern }}">Click Here</a> to add this project instead. </p>
```

Comment:
Ok. I'll change it. Thank you. 

    On Sunday, 31 January 2016 4:13 PM, Pierre-Yves Chibon <notifications@github.com> wrote:
 

 In anitya/templates/search.html:> @@ -88,6 +88,7 @@
>      <blockquote>
>          Oups this is embarrassing, it seems that no projects are being
>          monitored currently.
> +        <p><a href="/project/new?name={{ pattern }}">Click Here</a> to add this project instead. </p>
You need to use the url_for() construct from flask as otherwise it would break if anitya isn't deployed at the root of your domain name—
Reply to this email directly or view it on GitHub.